### PR TITLE
Require boost 1.64

### PR DESCRIPTION
**Pull request by rbx**
Bumping boost to 1.64, which includes Boost.Process library.

### CODE REVIEW

File: boost.sh [hunk 1]
```diff
@@ -1,18 +1,18 @@
 package: boost
 version: "%(tag_basename)s%(defaults_upper)s"
 source: https://github.com/alisw/boost.git
-tag: v1.59.0
+tag: v1.64.0
```
Comment (dennisklein) [inline boost.sh:4]: It is installed from alisw repo. Make sure they have updated it to recent boost.

Comment (rbx) [reply boost.sh:4]: Ah yeah. I will make an issue there if they can update it.

**Comment by fuhlig1** [comment]
Fixed with 2bf5f644e57496a2c72fc39fa228ed37696575f8

**Comment by rbx** [comment]
Does it make more sense to try to submit this kind of change to alidist directly? Project like O2 uses FairMQ, but also uses alidist, not alfadist...

**Comment by dennisklein** [comment]
Well, they will need to update, if FairRoot changes its deps. @fuhlig1 the recpie you merged will not work at the moment, because the alisw/boost repo does not contain version 1.64.

**Pull request closed without merging**